cfme/tests/networks/test_network_providers.py

```diff
@@ -3,14 +3,16 @@
 from widgetastic.exceptions import MoveTargetOutOfBoundsException
 
 from cfme.utils import error
+from cfme.utils.blockers import BZ
 from cfme.common.provider_views import NetworkProvidersView
 from cfme import test_requirements
 from cfme.networks.provider.nuage import NuageProvider, NetworkProvider
 
 pytestmark = [
     pytest.mark.provider([NetworkProvider], scope="module"),
     test_requirements.discovery,
-    pytest.mark.ignore_stream('5.8')
+    pytest.mark.ignore_stream('5.8'),
+    pytest.mark.uncollectif(BZ(1518301, forced_streams=['5.9']).blocks)
```

Comment:
**REQUIRED**
Please `@pytest.mark.meta`. We shouldn't perform any http requests during the test collection.